src/packages/@workspaces/common/hoc/WithButtonEnvironment.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  import { ButtonV2 } from "@library/ui";
+  /**
+   * icon to be embedded in button
+   */
+  export let icon;
+  /**
+   * function to invoke in button click
+   */
+  export let onClick;
+  /**
+   * identifies the button is desabled or enabled
+   */
+  export let disable;
+  /**
+   * identifies the button is in loading state or not
+   */
+  export let loader;
+  $: ({ onClick, icon, disable, loader } = $$props);
+</script>
+
+<ButtonV2
```

Comment:
Rename this file in format like  `WithButtonV3`  or `WithButtonV4`

Reply:
Resolved